java/src/main/java/ai/rapids/cudf/Scalar.java

```diff
@@ -484,6 +499,15 @@ public String getJavaString() {
     return getUTF8(getScalarHandle());
   }
 
+  /**
+   * Returns the scalar value as a column view.
+   * Callers should close the column view to avoid memory leak.
```

Comment:
The Javadoc needs to explain that the ColumnView **must not** outlive the Scalar because the view does not own the underlying GPU memory.  It is the responsibility of the caller to ensure the relative object lifetimes.

Reply:
Good suggestion, updated.